charts/sqlbackup/templates/_helpers.tpl

```diff
@@ -69,9 +69,9 @@ Back up script
 #!/bin/bash
 set -e
   {{- if or .Values.existingSecret .Values.existingSecretKey .Values.googleServiceAccount }}
-gcloud auth activate-service-account {{ .Values.googleServiceAccount }} --key-file=/secrets/existingSecret
+/usr/local/google-cloud-sdk/bin/gcloud auth activate-service-account {{ .Values.googleServiceAccount }} --key-file=/secrets/existingSecret
```

Comment:
Seems like `gcloud` should be on the PATH for the dsde-toolbox container :(

Reply:
I agree. Its been like this since forever. Maybe I'll make a PR into it...